src/shared/bg_pmove.cpp

```diff
@@ -4431,75 +4431,41 @@ PM_Animate
 */
 static void PM_Animate()
 {
-	if ( PM_Paralyzed( pm->ps->pm_type ) )
+	if ( PM_Paralyzed( pm->ps->pm_type )
+			|| pm->ps->tauntTimer > 0
+			|| pm->ps->torsoTimer != 0 )
 	{
 		return;
 	}
 
-	if ( usercmdButtonPressed( pm->cmd.buttons, BTN_GESTURE ) )
+	bool doit = false;
+	if ( IsSegmentedModel( pm->ps ) )
 	{
-		if ( pm->ps->tauntTimer > 0 )
+		if ( usercmdButtonPressed( pm->cmd.buttons, BTN_GESTURE ) )
 		{
-			return;
+			int wp_anim = TORSO_GESTURE_BLASTER + ( pm->ps->weapon - WP_BLASTER );
```

Comment:
Nit: our style for variable names is `wpAnim`

Reply:
exact.